source/Calamari.Tests/Fixtures/PowerShell/PowerShellFixture.cs

```diff
@@ -18,15 +18,34 @@ namespace Calamari.Tests.Fixtures.PowerShell
 {
     [TestFixture]
     [Category(TestCategory.CompatibleOS.Windows)]
-    public class PowerShellCoreOnWindows : PowerShellFixture
+    public class WindowsPowerShellCoreFixture : PowerShellFixture
```

Comment:
I think the previous name was better. The term `Windows PowerShell` has a very specific meaning (i.e. the traditional full .net framework version of powershell), which makes `WindowsPowerShellCore` very ambiguous.